linera-rpc/src/grpc/conversions.rs

```diff
@@ -631,22 +630,6 @@ impl TryFrom<api::PublicKey> for ValidatorPublicKey {
     }
 }
 
-impl From<ValidatorName> for api::PublicKey {
-    fn from(validator_name: ValidatorName) -> Self {
-        Self {
-            bytes: validator_name.0 .0.to_vec(),
-        }
-    }
-}
-
-impl TryFrom<api::PublicKey> for ValidatorName {
```

Comment:
will have to be restored soon?

Reply:
Not for `ValidatorName`. We will need to differentiate in proto between account's `PublicKey` and validator's `PublicKey`.

Reply:
Actually, let me do that right away - in this PR.

Reply:
Done.